mesmerize_viz/_store_model.py

```diff
@@ -1,15 +1,181 @@
+from multiprocessing.managers import Value
 from typing import *
 import numpy as np
-from fastplotlib import ImageGraphic, LinearSelector, ScatterGraphic, ImageWidget
-from ipywidgets import IntSlider, FloatSlider
+import pygfx
+from pygfx import PointerEvent
 
+from fastplotlib import ImageGraphic, LinearSelector, ScatterGraphic, ImageWidget, LineCollection, Figure
+from ipywidgets import IntSlider, FloatSlider, BoundedIntText
+
+from fastplotlib.graphics._collection_base import CollectionFeature
 from fastplotlib.graphics._features import FeatureEvent
+from fastplotlib.layouts._subplot import Subplot
+from fastplotlib.utils import get_nearest_graphics_indices
```

Comment:
this won't be in utils until the next release, just copy paste that function here until then?